mps_youtube/commands/play.py

```diff
@@ -92,7 +84,21 @@ def play(pre, choice, post=""):
                 streams.preload(g.model[chosen + 1], override=override)
 
         try:
-            play_range(songlist, shuffle, repeat, override)
+            if not config.PLAYER.get or not util.has_exefile(config.PLAYER.get):
+                g.message = "Player not configured! Enter %sset player <player_app> "\
+                    "%s to set a player" % (c.g, c.w)
+                return
+            else:
+                if config.PLAYER.get.endswith("mpv.exe"):
+                    mpv = Mpv()
+                    mpv.play_range(songlist, shuffle, repeat, override)
+                elif config.PLAYER.get.endswith("mplayer.exe"):
+                    mplayer = Mplayer()
+                    mplayer.play_range(songlist, shuffle, repeat, override)
+                else:
+                    g.message = "Player not supported! Enter %sset player <player_app> "\
```

Comment:
We are going to need some help on that. As you said, vlc is not fully integrated. Since vlc was using play_range in order to work, perhaps we can create another subclass for vlc, similar to the mpv and mplayer ones(with proper additions/deletions), and then improvements can be integrated there if someone wants to do that. Still, this would only work for vlc, if another player is used(other than the 3 "supported" ones) then the "else" part should remain the same. So let me ask:
1)Do you agree with this approach or is there a different way you would prefer? 
2)Are other players partially supported? Because, other than mpv and mplayer which are fully supported, and vlc, which is partially supported, we have not yet encountered such a case. If so, is this something you want? Shouldn't we try and integrate as many players as possible via subclasses instead of relying on the current program to work under `set player unintegrated_player_name` ?